packages/grafana-data/src/types/datasource.ts

```diff
@@ -517,6 +517,7 @@ export interface QueryFix {
 
 export interface QueryFixAction {
   type: string;
+  hint?: string;
```

Comment:
I am wondering, if we should add here different name of property. Because whole thing is `Query hint`, so `hint` might be generic. Maybe `title`, `name`?

Reply:
It was actually my suggestion. Label has been already used, twice, for the description of the hint.

![imagen](https://user-images.githubusercontent.com/1069378/207607429-dfc3318a-fb99-4be5-a46a-39f58b8bb695.png)

I suggested `hint` because it's what we end up showing.

Reply:
Looking at this more, I am wondering if we should add hint's name/title/hint up next to the label. Cause name is not relevant to the action. 
      ```
      hints.push({
        type: 'ADD_LOGFMT_PARSER',
        label: 'Selected log stream selector has logfmt formatted logs.',
        fix: {
          label: 'Consider using logfmt parser to turn key-value pairs in your log lines to labels.',
          title: 'add logfmt parser';
          action: {
            type: 'ADD_LOGFMT_PARSER',
            query,
          },
        },
      });
      ```

Reply:
+1